src/test/java/uk/gov/ons/ctp/integration/envoycuc/client/RateLimiterClientRequest.java

```diff
@@ -8,33 +8,11 @@
 import uk.gov.ons.ctp.integration.ratelimiter.client.RateLimiterClient.Domain;
 
 @Data
-public class RateLimiterClientRequest {
-
-  private RateLimiterClient.Domain domain = Domain.RH;
-  private Product product;
-  private CaseType caseType;
-  private String ipAddress;
-  private UniquePropertyReferenceNumber uprn;
-  private String telNo;
-
-  public RateLimiterClientRequest(
-      Domain domain,
-      Product product,
-      CaseType caseType,
-      String ipAddress,
-      UniquePropertyReferenceNumber uprn,
-      String telNo) {
-    this.domain = domain;
-    this.product = product;
-    this.caseType = caseType;
-    this.ipAddress = ipAddress;
-    this.uprn = uprn;
-    this.telNo = telNo;
-  }
-
-  public RateLimiterClientRequest() {}
-
-  public String toString() {
-    return String.format(product.toString(), caseType, ipAddress, uprn, telNo);
-  }
+public abstract class RateLimiterClientRequest {
```

Comment:
Can't see why this class needs to have all the same fields as RateLimiterClientFulfilmentRequest.
Are the superclass fields actually used?

Reply:
The fields are all protected in the base class - maybe I changed it (this monring) after you'd first looked.

Reply:
Resolved. Details covered by next conversation